static/js/giphy.js

```diff
@@ -7,11 +7,31 @@ import render_giphy_picker from "../templates/giphy_picker.hbs";
 import render_giphy_picker_mobile from "../templates/giphy_picker_mobile.hbs";
 
 import * as compose_ui from "./compose_ui";
+import {media_breakpoints} from "./css_variables";
 import {page_params} from "./page_params";
+import * as popovers from "./popovers";
 import * as ui_util from "./ui_util";
 
 const giphy_fetch = new GiphyFetch(page_params.giphy_api_key);
 let search_term = "";
+let active_popover_element;
+
+// Only used if popover called from edit message, otherwise it is `undefined`.
+let edit_message_id;
+
+export function is_popped_from_edit_messsage() {
+    return edit_message_id !== undefined;
```

Comment:
I changed this to also check `active_popover_element`, mainly so if we try to write `is_popped_from_compose` by copying the existing logic, we'll do it right.